src/models/soil/soil_model_bcs.jl

```diff
@@ -50,12 +65,27 @@ ImpermeableBoundary() = NoFlux(:saturation_water_ice)
 Alias for `PrescribedGradient` representing a Neumann-type zero pressure gradient at the bottom of the soil
 column, thereby allowing free drainage of water.
 """
-FreeDrainage(::Type{NF}) where {NF} = PrescribedGradient(:water_potential, zero(NF))
+FreeDrainage(::Type{NF}) where {NF} = PrescribedGradient(:pressure_head, zero(NF))
+
+# SoilBoundaryConditions constructor
 
 """
-Alias for `ColumnBoundaryConditions` with defaults suitable for `SoilModel`s.
+Creates `ColumnBoundaryConditions` with defaults suitable for the given soil process configurations.
 """
-SoilBoundaryConditions(::Type{NF}; top=default_soil_upperbc(NF), bottom=default_soil_lowerbc(NF)) where {NF} = ColumnBoundaryConditions(; top, bottom)
+SoilBoundaryConditions(
```

Comment:
I find it a bit confusing that there is `SoilBC` and `SoilBoundaryConditions`. The former is just one boundary, correct? And `SoilBoundaryConditions` just wraps around it to initialise top and bottom boundary conditions for one column.  

What's actually also our convention of abbreviations in types? I am rather in favour of writing out abbreviations in types. `SoilBoundaryCondition` and `SoilBoundaryConditions` are obviously also very easy to mistake for each other though.